### PR TITLE
Added Japanese Translation

### PR DESCRIPTION
**Pull request by yoshi818**
Hello I'd made Japanese translation for your LSPosed.
If you like it, please pull this.

### CODE REVIEW

File: app/src/main/res/values-ja/strings.xml [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+    <string name="module_release_no_more">これ以上のリリースはありません</string>
+    <string name="repo_load_failed">モジュールリポジトリの読み込みに失敗: %s</string>
+
+    <string name="app_destroyed">本アプリケーションの内部で壊れています。公式ソースからダウンロードされたものであるかを確認して下さい。</string>
+    <string name="outdated_manager">LSPosed Manager と LSPosed Core のバージョンが一致していません。対応バージョンを再インストールして下さい。</string>
+    <string name="lsposed_not_active">LSPosed が未インストールもしくは有効化されていません。</string>
+    <string name="verbose_log_not_avaliable">詳細ログが存在しません。ログ出力有効化に再起動を行っていない場合、再起動して下さい。</string>
+    <string name="NotInstall">未インストール</string>
+    <string name="NotInstallDetail">LSPosed が未インストールです</string>
+    <string name="translators"><![CDATA[<a href="https://github.com/LSPosed/LSPosed">LSPosed</a>]]></string>
```
Comment (yujincheng08) [inline app/src/main/res/values-ja/strings.xml:190]: ここに名前を入れてください。

**Review by yujincheng08 — CHANGES_REQUESTED** [review]

File: app/src/main/res/values-ja/strings.xml (hunk 1 above)
Comment (yujincheng08) [reply app/src/main/res/values-ja/strings.xml:190]: ホームページと一緒に

File: app/src/main/res/values-ja/strings.xml [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+    <string name="module_installed">ユーザ %2$s へ %1$s を追加</string>
+    <string name="module_install_failed">モジュールの追加に失敗</string>
+    <string name="install_to_user">ユーザ %s でのインストール</string>
+    <string name="install_to_user_message">ユーザ %2$s へ %1$s をインストールしますか?手動でのインストールをお勧めします。LSPosed 経由での強制インストールでは、問題が発生する可能性があります。</string>
+
+    <!-- AppListActivity -->
+    <string name="compile_speed">最適化</string>
+    <string name="compile_speed_msg">最適化中…</string>
+    <string name="compile_done">最適化を完了しました。</string>
+    <string name="app_launch">Launch it</string>
```
Comment (yujincheng08) [inline app/src/main/res/values-ja/strings.xml:102]: 起動します？

File: app/src/main/res/values-ja/strings.xml [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+    <string name="menu_sort">ソート中…</string>
+    <string name="enable_module">モジュールの有効化</string>
+    <string name="no_scope_selected">アプリが未選択です。継続しますか?</string>
+    <string name="menu_show_games">ゲーム</string>
+    <string name="menu_show_modules">モジュール</string>
+    <string name="failed_to_save_scope_list">スコープリストの保存に失敗</string>
+    <string name="app_description">%1$s\nバージョン %2$s</string>
+    <string name="use_recommended">お勧め</string>
+    <string name="no_scope_selected_has_recommended">アプリが未選択です。お勧めアプリを選択しますか?</string>
+    <string name="use_recommended_message">Select recommended apps?</string>
```
Comment (yujincheng08) [inline app/src/main/res/values-ja/strings.xml:123]: 未翻訳

File: app/src/main/res/values-ja/strings.xml [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+    <string name="android_framework">システムフレームワーク</string>
+    <string name="menu_backup_and_restore">バックアップ…</string>
+    <string name="menu_backup">バックアップ</string>
+    <string name="menu_restore">レストア</string>
+    <string name="force_stop">強制停止</string>
+    <string name="force_stop_dlg_title">強制停止しますか?</string>
+    <string name="force_stop_dlg_text">アプリを強制停止すると、システムが誤動作する可能性があります。</string>
+    <string name="reboot_required">本変更適用後に再起動が必要です</string>
+    <string name="reboot">再起動</string>
+    <string name="menu_hide">隠す…</string>
```
Comment (yujincheng08) [inline app/src/main/res/values-ja/strings.xml:136]: 排除？

**Comment by yoshi818** [comment]
yujincheng08
Thank you for your reply.I've fixed your suggestion.

**Review by yujincheng08 — APPROVED** [review]

**Comment by yujincheng08** [comment]
thx so much. merged

**Comment by yujincheng08** [comment]
Hi contributor, we are here inviting you to use the localization manage platform Crowdin for translations. Please check: https://lsposed.crowdin.com/lsposed

**Pull request merged**